Cargo.toml

```diff
@@ -59,6 +59,7 @@ rover-http = { path = "./crates/rover-http" }
 rover-graphql = { path = "./crates/rover-graphql" }
 rover-std = { path = "./crates/rover-std" }
 rover-studio = { path = "./crates/rover-studio" }
+rover-subtask = { path = "./crates/rover-subtask" }
```

Comment:
I'm not sure if we need to pull this out, since it's only `rover` that uses this module/crate.